app/src/main/res/values-es/strings-es.xml

```diff
@@ -359,124 +358,317 @@
 	<string name="parakeet_and_extra_features">Parakeet y características adicionales</string>
 	<string name="insulin_duration_hours">Duración de la insulina en horas</string>
 	<string name="default_liver_sensitivity_ratio">Relación de sensibilidad por defecto de hígado</string>
-	<string name="key_is_used_instead_of_google_account">Clave se utiliza en lugar de la cuenta de Google</string>
-	<string name="sync_using_custom_security_key">Sincronizar utilizando la clave de seguridad personalizada</string>
-	<string name="handset_group_security_key">Grupo microteléfono sincronización de seguridad clave</string>
+	<string name="key_is_used_instead_of_google_account">Clave utilizada en lugar de la cuenta de Google</string>
+	<string name="sync_using_custom_security_key">Sincronizar utilizando clave de seguridad personalizada</string>
+	<string name="handset_group_security_key">Clave Seguridad Grupo sincronización</string>
 	<string name="handset_sync_grouping_key">Auricular clave de sincronización agrupación</string>
 	<string name="alert_name_colon">Nombre de alerta:</string>
 	<string name="threshold_colon">Límite:</string>
 	<string name="default_snooze_colon">Repetir por defecto:</string>
-	<string name="re_raise_every_x_minutes_if_unaknowledged">Volver a subir cada x minutos si\n\nunaknowledged:</string>
+	<string name="re_raise_every_x_minutes_if_unaknowledged">Volver a disparar cada x minutos si\n\nunaknowledged:</string>
 	<string name="alert_tone_colon">Tono de alerta:</string>
 	<string name="choose_file">Elija el archivo</string>
 	<string name="select_time_for_alert_colon">Seleccione el tiempo para la alerta:</string>
-	<string name="all_day">todo el dia</string>
+	<string name="all_day">todo el día</string>
 	<string name="tap_to_change">(Pulsar para cambiar)</string>
-	<string name="override_phone_silent_mode_colon">Teléfono anulación del modo silencioso:</string>
+	<string name="override_phone_silent_mode_colon">Anulación del modo silencioso del teléfono:</string>
 	<string name="vibrate_on_alert">Vibrar en alerta</string>
-	<string name="test_alert">probar alerta</string>
-	<string name="save_alert">guardar alerta</string>
-	<string name="remove_alert">eliminar alerta</string>
+	<string name="test_alert">Probar alerta</string>
+	<string name="save_alert">Guardar alerta</string>
+	<string name="remove_alert">Eliminar alerta</string>
 	<string name="snooze_alert_before_it_fires">Silenciar alerta antes del disparo</string>
 	<string name="create_low_alert">Crear alerta de baja</string>
 	<string name="create_high_alert">Crear alerta de alta</string>
 	<string name="long_press_an_existing_alert_to_edit">(Pulsación larga para editar una alerta existente)</string>
 	<string name="adding">Añadiendo</string>
 	<string name="high">alto</string>
 	<string name="alert">alerta</string>
-	<string name="high_alert">Alerta alta</string>
-	<string name="low_alert">Alerta de baja</string>
+	<string name="high_alert">Alerta hiperglucemia</string>
```

Comment:
outside the alarm limits is not always Hyperglycemia / Hypoglycemia but High / low